src/main/groovy/org/arquillian/spacelift/gradle/utils/KillJavas.groovy

```diff
@@ -24,6 +26,42 @@ class KillJavas extends Task<Object, Void>{
 
     def processPorts = [4444, 14444, 8080, 9999]
 
+    KillJavas addProcessNames(List<String> processNames) {
+        if (processNames && !processNames.isEmpty()) {
+            this.processNames.addAll(processNames)
+        }
+        this
+    }
+
+    KillJavas addProcessName(String processName) {
+        if (processName) {
+            processNames.add(processName)
+        }
+        this
+    }
+
+    KillJavas addProcessPort(int port) {
+        if (port > 1024 && port < 65535) {
+            processPorts.add(port)
```

Comment:
This does not makes sense.

First, you might be running with enough privileges to kill process bound to a privileged port. Second, you should not ignore user adding something without notice.

If you want to keep this check, please emit a warning that privileged port was added and task might fail